src/forms/submission-handlers/base-handlers.ts

```diff
@@ -68,7 +68,11 @@ export const ObsSubmissionHandler: SubmissionHandler = {
         );
       }
       if (typeof obs.value == 'string' || typeof obs.value == 'number') {
-        return field.questionOptions.rendering == 'date' ? moment(obs.value).toDate() : obs.value;
+        if (field.questionOptions.rendering == 'date') {
+          field.value.value = moment(field.value.value).format('YYYY-MM-DD HH:mm');
+          return moment(obs.value).toDate();
```

Comment:
whats happening here?

Reply:
Will be fixed with the update of the rebase